db/seeds.rb

```diff
@@ -1,109 +1,144 @@
 USERS_TO_CREATE = 20
-USERS_TO_CREATE.times do
-
-	User.create company:					Faker::Company.name,
-							email:						Faker::Internet.email,
-							password:	        Faker::Internet.password,
-							employees:				rand(8),
-							website:					Faker::Internet.url
+NUMBER = 0
 
+USERS_TO_CREATE.times do
+  NUMBER += 1
+  User.create(company: Faker::Company.name + NUMBER.to_s,
+              email: NUMBER.to_s + Faker::Internet.email,
+              password: Faker::Internet.password,
+              employees: rand(8),
+              website: Faker::Internet.url + NUMBER.to_s)
 end
-# works
 
-User.create company:					"Admin Man",
-						email:						"admin@admin.com",
-						password:	        "123456",
-						employees:				rand(8),
-						website:					Faker::Internet.url,
-						admin:						true
+CREATE_ONE_ADMIN = 1
```

Comment:
Similar variables prior were named with nouns e.g. "users" . This one starts with a verb. Not a big deal but strays away from the file's convention.